### PR TITLE
Update jira-card-creator

### PR DESCRIPTION
**Pull request by baconYao**
Update jira-card-creator readme and add workflow
    
    1. Update the README file of jira-card-creator
    2. Add test-jira-card-creator github workflow
    3. Mock the dependency so we can perform the pure unit test dirctly
    4. Enhance the setup.sh so that we can copy the dependent APIs

### CODE REVIEW

**Comment by baconYao** [comment]
The CI test in my personal repo: https://github.com/baconYao/oem-qa-tools/actions/runs/4663043756/jobs/8254012201

**Review by weizhenwu — APPROVED** [review]
LGTM

File: Tools/PC/jira-card-creator/README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ This tool creates card based on Project Books.
 
 ## How To Use
 
-This tool is depend on [Jira](https://github.com/canonical/oem-qa-tools/blob/main/API/Jira) and [GoogleSheet](https://github.com/canonical/oem-qa-tools/blob/main/API/GoogleSheet) APIs services. Please `Copy` these two APIs to this directory, you will see the following structure before you start.
+This tool is depend on [Jira](https://github.com/canonical/oem-qa-tools/blob/main/API/Jira) and [GoogleSheet](https://github.com/canonical/oem-qa-tools/blob/main/API/GoogleSheet) APIs services.
+
+Please see [example sextion](#example) to learn how to use it.
```
Comment (stanley31huang) [inline Tools/PC/jira-card-creator/README.md:11]: typo here, `section`

Comment (baconYao) [reply Tools/PC/jira-card-creator/README.md]: Fix it, thanks.

**Pull request merged**